junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala

```diff
@@ -161,7 +161,7 @@ final class JUnitExecuteTest(task: JUnitTask, runSettings: RunSettings,
   }
 
   private def emitMethodEvent(methodName: String, status: Status): Unit = {
-    val selector = new NestedTestSelector(task.taskDef.fullyQualifiedName, methodName)
+    val selector = new TestSelector(task.taskDef.fullyQualifiedName + "." + runSettings.decodeName(methodName))
```

Comment:
Is that a bug fix? If so, it would be good if it could have its own commit message, or at least be mentioned in the commit message.